src/index.js

```diff
@@ -66,7 +66,7 @@ class DBLAPI extends EventEmitter {
         }, this.options.statsInterval);
       }
 
-      if (client.readyAt !== null || client.startTime !== null) {
+      if (client.ready || client.ws.status === 0) {
```

Comment:
Might cause crash on eris since it doesnt have a `ws` property

Reply:
I was thinking that client.ready would already exist, thinking about it with more than half a brain cell, yeah. Alright.
By the way by the end of all these improvements I'm gonna close it and create a new PR to not fill up the commits.

Reply:
How would you suggest fixing this?
Should I check the library,

or should I use like `client.ready || (client.ws || {}).status === 0`
I would use `?.` but that's only available in v14

Reply:
> By the way by the end of all these improvements I'm gonna close it and create a new PR to not fill up the commits.

I'm going to do a squash merge, don't worry about that

> How would you suggest fixing this?

`client.ws && client.ws.status`

Reply:
> I'm going to do a squash merge, don't worry about that

Sounds good